en/docs/extended_topics/how_to_convert_image_to_ofrecord.md

```diff
@@ -0,0 +1,147 @@
+# Convert the image files to OFRecord datasets
+
+In  [OFRecord Data Format](./ofrecord.md) and  [Loading and Preparing OFRecord Dataset](./how_to_make_ofdataset.md), We learned how to convert other dataset formats to OFRecord datasets separately and how to load OFRecord datasets.
+
+In this article, we will explain how to make image files into OFRecord datasets. Also we provide relevant script for users to use directly or make modification base on that. Which includes: 
+
+- OFRecord datasets based on MNIST Handwritten Digits.
+
+- How OFRecord Reader is encoded.
+- Training on OFRecord dataset.
+
+### OFRecord datasets based on MNIST Handwritten Digits.
+
+We use MNIST Handwritten Digits dataset to completely produce an OFRecord format file.
+
+we only take 50 pictures for demonstration. The relevant script and datasets please refer to [img2ofrecord](https://oneflow-static.oss-cn-beijing.aliyuncs.com/oneflow-tutorial-attachments/img2ofrecord.zip).
+
+- Download and unzip the relevant zip file
+
+```shell
+$ wget https://oneflow-static.oss-cn-beijing.aliyuncs.com/oneflow-tutorial-attachments/img2ofrecord.zip
+$ unzip img2ofrecord.zip
+```
+
+- Go to the corresponding directory and run OFRecord production script `img2ofrecord.py`
```

Comment:
Change directory to corresponding path and run OFRecord production script `img2ofrecord.py`.